doc-General_Configuration/topics/Configuration.adoc

```diff
@@ -2690,13 +2678,9 @@ The following is an example of regionalized database scenario:
 * Enable only the *Reporting*, *Scheduler*, and *User Interface Server* Roles.
 To perform database maintenance items, such as scheduled backups, on the top-level region (master), also enable the *Database Operations* role.
 * No additional settings aside from assigning the region ID. No need to configure any replication.
-. *Create Region Number 1*
-.. Add replication worker settings pointing to the VMDB for *Region 99*.
-.. Enable *Database Synchronization Server* role on one *Server* in the *Region*. If you have a second *Server* in the same region, do not enable the *DB Synchronization* role.
-Do not enable more than one *Database Synchronization Role* per Region.
-. *Create Region Number 2*
-.. Add replication worker settings pointing to the VMDB for *Region 99*.
-.. Enable *Database Synchronization Server* role on one *Server* in the *Region*. If you have a second *Server* in the same region, do not enable the *DB Synchronization* role. Do not enable more than one *Database Synchronization Role* per Region.
+. *Create Region Number 1*.
+. *Create Region Number 2*.
+
```

Comment:
I'm not sure how helpful this is ... Maybe the entire section starting at https://github.com/ManageIQ/manageiq_docs/pull/200/files#diff-a6c5c5812c8fd47efcd7d46d7c12b918R2674 could be removed?

If we were to add any more detail we would probably just end up with a copy of the section about [Configuring a Remote Copy](https://github.com/ManageIQ/manageiq_docs/pull/200/files#diff-a6c5c5812c8fd47efcd7d46d7c12b918R2735)